KIPs/kip-fungible_token.md

```diff
@@ -0,0 +1,240 @@
+---
+kip: 7
+title: Fungible Token Standard
+author: Junghyun Colin Kim <colin.kim@groundx.xyz>, Kyungup Kim <ethan.kim@groundx.xyz>
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Token
+created: 2020-02-20
+---
+
+<!--You can leave these HTML comments in your merged KIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new KIPs. Note that a KIP number will be assigned by an editor. When opening a pull request to submit your KIP, please use an abbreviated title in the filename, `kip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the KIP.-->
+A fungible token standard for Klaytn.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+The following standard allows for the implementation of a standard API for tokens within smart contracts.
+This standard provides basic functionality to transfer tokens.
+
+## Motivation
+<!--The motivation is critical for KIPs that want to change the Klaytn protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the KIP solves. KIP submissions without sufficient motivation may be rejected outright.-->
+A standard interface allows any token on Klaytn to be re-used by other applications: from wallets to decentralized exchanges.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Klaytn platforms (klaytn). -->
+This document derived heavily from Ethereum's [ERC-20 token standard](https://eips.ethereum.org/EIPS/eip-20) written by Fabian Vogelsteller and Vitalik Buterin.
+Comparing with ERC-20 token standard, this token standard includes mint/burn functions and enforces the triggering of transfer events when minting / burning tokens. 
+
+### Summary of Methods and Events
+The table below is a summary of methods.
+The prototype uses the syntax from Solidity `0.4.24` (or above).
+If the optional field is not marked, the function must be implemented.
+
+|Name|Optional|Prototype|
+|---|:---:|---|
+|[name](#name)|O|function name() public view returns (string)|
```

Comment:
Can we merge those optional functions together? top or bottom as you want.

Reply:
I think "transfer" is more import than "mint/burn/burnFrom" therefore his order was sorted by importances of the items even if there are optional items. 
So I think this present order is ok.
Do you want me to sort by the option rather than importance? ?

Reply:
What about `name` and `transfer`? Do you think `name` is more important than `transfer`?

Reply:
Above function are view functions. And Below functions are not view function.
I think they are sorted by if it is view function or not before sorting importance. :)